src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioning.scala

```diff
@@ -0,0 +1,76 @@
+package org.hammerlab.guacamole.loci.partitioning
+
+import java.io.{InputStream, OutputStream}
+
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.rdd.RDD
+import org.hammerlab.guacamole.loci.map.LociMap
+import org.hammerlab.guacamole.loci.partitioning.LociPartitioner.PartitionIndex
+import org.hammerlab.guacamole.loci.set.LociSet
+import org.hammerlab.guacamole.reference.{Locus, ReferenceRegion}
+import org.hammerlab.magic.util.{Saveable, Stats}
+
+import scala.reflect.ClassTag
+
+case class LociPartitioning(map: LociMap[PartitionIndex]) extends Saveable {
+  @transient lazy val partitionsMap: Map[PartitionIndex, LociSet] = map.inverse
```

Comment:
A comment about why these are `@transient lazy val` would be useful (guessing it's serialization?) Also a class-level docstring with a sentence for what this class represents would be helpful